spec/lucky/params_spec.cr

```diff
@@ -148,4 +184,85 @@ describe Lucky::Params do
       params.nested(:missing).should eq({} of String => String)
     end
   end
+
+  describe "get_file" do
+    it "gets files" do
+      request = build_multipart_request file_parts: {
+        "welcome_file" => "welcome file contents",
+      }
+
+      params = Lucky::Params.new(request)
+
+      file = params.get_file!(:welcome_file)
+      file.is_a?(Tempfile).should eq(true)
+      File.read(file.path).should eq "welcome file contents"
+    end
+
+    it "gets files alongside params" do
+      request = build_multipart_request(
+        form_parts: {
+          "from" => "multipart",
+        },
+        file_parts: {
+          "with" => "a file",
+        }
+      )
+
+      params = Lucky::Params.new(request)
+
+      file = params.get_file!(:with)
+      File.read(file.path).should eq "a file"
+    end
+
+    it "raises if missing a param and using get_file! version" do
+      request = build_multipart_request form_parts: {"this" => "that"}
+
+      params = Lucky::Params.new(request)
+
+      expect_raises do
```

Comment:
In Crystal 0.24 I believe you need to pass an exception type. Can you add an exception type to this? Something like `expect_raises WhateverTheExceptionIs`

Reply:
They're just generic `Exception` at the moment but I've added that and the error messages.